### ISSUE TITLE
support --help for oxidation-compiler

### ISSUE THREAD

**Issue by hardfist** [Boshen/oxc#398]

**Comment by Boshen** [comment]
What's not working?

# `npx oxidation-compiler --help`

```
The JavaScript Oxidation Compiler

Usage: oxc <COMMAND>

Commands:
  lint  Lint this repository.
  help  Print this message or the help of the given subcommand(s)

Options:
  -h, --help     Print help
  -V, --version  Print version
```

# `npx oxidation-compiler lint --help`

```
Lint this repository.

Usage: oxc lint [OPTIONS] <PATH>...

Arguments:
  <PATH>...  File or Directory paths to scan. Directories are scanned recursively.

Options:
      --fix
          This option allows you to enable oxc to fix as many issues as possible. If enabled, only unfixed issues are reported in the output
      --quiet
          This option allows you to disable reporting on warnings. If you enable this option, only errors are reported by oxc_lint.
      --ignore-path <ignore-path>
          This option allows you to specify the file to use as your .eslintignore.
      --no-ignore
          Disables excluding of files from .eslintignore files, --ignore-path flags, --ignore-pattern flags.
      --ignore-pattern <ignore-pattern>
          This option allows you to specify patterns of files to ignore (in addition to those in .eslintignore).
      --max-warnings <max-warnings>
          This option allows you to specify a warning threshold, which can be used to force oxc_lint to exit with an error status if there are too many warning-level rule violations in your project.
  -h, --help
          Print help
```

**Comment by hardfist** [comment]
![image](https://github.com/Boshen/oxc/assets/8898718/321273ad-12ae-4dbd-b151-991461fb96e8)
